projects/noloan/common/src/main/proto/Sms.proto

```diff
@@ -11,9 +13,11 @@ message SmsMessage {
     string receivedAt = 3;
     string id = 4;
     bool approved = 6;
+    Company company = 7;
```

Comment:
We don't renumber proto fields that are stored in binary form. `SmsMessage` is stored in binary form in the DB.
If you do that, you will get weird errors.

So please leave the field numbers as they are.

Reply:
Didn't think of that.... :+1:

Reply:
I deleted the messages in the DB - Starting over with this ordering.

Pushed it as it is...